src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py

```diff
@@ -215,6 +215,26 @@ def initTest(self):
                     'value': 'disable'
                 }
             },
+            'DEVICE_METADATA_CORRECT_BUFFER_MODEL_PATTERN': {
+                'desc': 'DEVICE_METADATA correct value for BUFFER_MODEL field',
+                'eStr': self.defaultYANGFailure['Verify'],
```

Comment:
Just Say self.defaultYANGFailure['None']

And Remove:
```
'verify': {'xpath': '/sonic-device_metadata:sonic-device_metadata/DEVICE_METADATA/localhost/buffer_model',
                    'key': 'sonic-device_metadata:buffer_model',
                    'value': 'dynamic'
```

We usually use verify for default statements in YANG.